runtime/common/src/migrations.rs

```diff
@@ -39,7 +39,7 @@ impl<T: ParachainStakingConfig> Migration
 	for ParachainStakingIncreaseMaxDelegationsPerCandidate<T>
 {
 	fn friendly_name(&self) -> &str {
-		"MM_Parachain_Staking_IncreaseMaxDelegationsPerCandidate"
+		"MM_Parachain_Staking_IncreaseMaxDelegationsPerCandidate_v2"
```

Comment:
I think this is a "clean" way to make this migration run again, specifically:

1. On chains where it has already run, it will exist in `pallet_migrations` storage as two different runs.
2. It will run again on these same chains.
3. It will run only once on chains where it has not run yet (and exist as the `_v2` variant in storage).